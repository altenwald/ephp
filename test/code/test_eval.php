<?php
eval("echo 'Inception lvl 1...\n'; eval('echo \"Inception lvl 2...\n\"; eval(\"echo \'Inception lvl 3...\n\'; eval(\'echo \\\"Limbo!\\\";\');\");');");
