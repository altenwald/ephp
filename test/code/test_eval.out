Inception lvl 1...
Inception lvl 2...
Inception lvl 3...
Limbo!